aws-ts-eks-hello-world/README.md

```diff
@@ -205,23 +225,21 @@ After cloning this repo, from this working directory, run these commands:
     in-place, and which require replacement, and computes
     the minimally disruptive change to achieve the desired state.
 
-    **Note:** Pulumi auto-generates a suffix for all objects. Pulumi's object model does
-    create-before-delete replacements by default on updates, but this will only work if
-    you are using name auto-generation so that the newly created resource is
-    guaranteed to have a differing, non-conflicting name. Doing this
-    allows a new resource to be created, and dependencies to be updated to
-    point to the new resource, before the old resource is deleted.
-    This is generally quite useful.
-
-    ```
-    ...
-
-    + deploymentName : "helloworld-58jkmc7c"
-    ...
-    + namespaceName  : "helloworld-xaldhgca"
-    + serviceHostname: "a71f5ab3f2a6e11e3ac39200f4a9ad5d-1297981966.us-west-2.elb.amazonaws.com"
-    + serviceName    : "helloworld-3fc2uhh7"
-    ```
+    > **Note:** Pulumi auto-generates a suffix for all objects. Pulumi's object model does
+    > create-before-delete replacements by default on updates, but this will only work if
+    > you are using name auto-generation so that the newly created resource is
+    > guaranteed to have a differing, non-conflicting name. Doing this
+    > allows a new resource to be created, and dependencies to be updated to
+    > point to the new resource, before the old resource is deleted.
+    > This is generally quite useful.
+    >
+    > ```
+    > deploymentName : "helloworld-58jkmc7c"
+    > ...
+    > namespaceName  : "helloworld-xaldhgca"
+    > serviceHostname: "a71f5ab3f2a6e11e3ac39200f4a9ad5d-1297981966.us-west-2.elb.amazonaws.com"
+    > serviceName    : "helloworld-3fc2uhh7"
+    > ```
```

Comment:
I think we can replace all this with just:

```
**Note:** Pulumi auto-generates a suffix for all objects.  For more details on why see [documentation on resource names](https://pulumi.io/reference/programming-model.html#names).
```

That linked page is being deployed right now: https://github.com/pulumi/docs/pull/871.

Reply:
👍